schema.graphql

```diff
@@ -33,3 +33,85 @@ type AuctionedName @entity {
   bidCount: Int!
   state: AuctionState
 }
+
+type Resolver @entity {
+  id: ID!                   # Concatenation of resolver address and namehash
+  domain: Domain!
+  resolverEvents: [ResolverEvent!]! @derivedFrom(field: "resolverID")
+}
+
+interface ResolverEvent {
+  id: ID!                   # Concatenation of block number and log ID
+  node: Bytes!
+  resolver: Bytes!          # Address of the resolver that emitted the event
```

Comment:
Wouldn't it make more sense to put this field on the Resolver entity, instead, and call it, eg, Address?

Reply:
yes will update